PowerEditor/src/Notepad_plus.cpp

```diff
@@ -2346,6 +2351,28 @@ void Notepad_plus::setLangStatus(LangType langType)
 	_statusBar.setText(getLangDesc(langType).c_str(), STATUSBAR_DOC_TYPE);
 }
 
+void Notepad_plus::setPercentStatus(int whichOne, int percent)

```

Comment:
Renaming parameter `whichOne` to `whichView` would make its purpose more clear IMO.

Reply:
What's your suggestion @MAPJe71 ?